src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx

```diff
@@ -1,24 +1,32 @@
 import React, { FunctionComponent, useState } from "react";
 import { useDropzone } from "react-dropzone";
-import { readFileAsJson } from "../../../common/utils";
+import { readFileAsJson, readFileAsCsv } from "../../../common/utils";
 import { Button } from "../../UI/Button";
 import { ErrorAlert } from "../../Alert";
 import { getLogger } from "../../../utils/logger";
+import { useFormsContext } from "../../../common/context/forms";
 
 const { stack } = getLogger("DataFileButton");
 interface DataFileButton {
   onDataFile: (dataFile: any) => void; // eslint-disable-line @typescript-eslint/no-explicit-any
 }
 
 export const DataFileButton: FunctionComponent<DataFileButton> = ({ onDataFile }) => {
+  const { currentFormTemplate } = useFormsContext();
   const [error, setError] = useState(false);
   const onDrop = async (files: File[]): Promise<void> => {
     try {
       const file = files[0];
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      const config = await readFileAsJson<any>(file);
+      let data = null;
+      if (file.name.indexOf(".csv") > 0) {
+        // has to be > 0...
```

Comment:
best comment :D

Reply:
hahah yeah I introduced a new bug, took me a while to find it... will remove it soon